### DISCUSSION TITLE
SES not working with attachment

### DISCUSSION THREAD

**Discussion by vvijayalakshmi21** [aws/aws-sdk-java#3044] [Q&A]
Hi,
I need to send an attachment (xlsx) using SES v1 and I followed the sample given [here](https://docs.aws.amazon.com/ses/latest/dg/send-email-raw.html)
Here is my observation:
- The mail is received when I comment out the part where I attach the file
- The mail is NOT received when I attach the file to the mail. However, **_no errors thrown_** and it says “Mail successfully sent”. I don't see the message count increased on Cloudwatch too.

From the console logs, I see that the attachment is present in the mail body.
What could be the possible reasons?

If an org policy prohibits attachments, would that be considered at AWS as success or failure? 
Is this tracked somewhere?

Please can someone help in this regard?

**Comment by debora-ito** [comment]
@vvijayalakshmi21 is the code you're running exactly like the one in the [SES doc](https://docs.aws.amazon.com/ses/latest/dg/send-email-raw.html#send-email-raw-api)? If not, can you share the code exactly as you are running it?
Please also share:
- The version of the SDK you're using
- What's the size of the xlsx file

**Reply by vvijayalakshmi21** [reply]
Yes. I had used exactly the same as in the sample.
I am on Java 17 & the size of the file is ~11 kB.

The root cause of the issue has been identified. 
The email ID being used for sending mail was not authorized to send attachments as per the org policy & hence quarantined.
If I use the one authorised to send attachments, the mail is sent successfully with the same code.

Can you please advise where I could find those mails (which are quarantined) on Cloudwatch?

**Reply by debora-ito** [reply]
I don't know if attachment authorization is logged on CloudWatch, I think it depends on how the authorization is managed. This is a question about SES and CloudWatch, which the SDK teams are not necessarily knowledgeable about.

In any case, we are glad you identified the root cause. Let us know if you have any other question about the Java SDK.
